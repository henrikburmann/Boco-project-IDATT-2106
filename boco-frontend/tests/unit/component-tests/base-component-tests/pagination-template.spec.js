import { mount } from "@vue/test-utils";
import Pagination from "@/components/BaseComponents/PaginationTemplate.vue";

describe("PaginationTemplate", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(Pagination, {
      //passing prop to component
      props: {
        items: [],
        currentPage: 0,
        pageSize: 4,
      },
    });
  });

  it("renders correctly", () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  it("is instantiated", () => {
    expect(wrapper.exists()).toBeTruthy();
  });
});
