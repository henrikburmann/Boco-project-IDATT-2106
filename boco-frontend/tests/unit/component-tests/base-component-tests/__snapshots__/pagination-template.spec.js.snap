// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginationTemplate renders correctly 1`] = `
<div
  data-v-app=""
>
  
  <!-- Pagination -->
  <!--v-if-->
  
</div>
`;
